### ISSUE TITLE
stop using abbreviations for default vm names

### ISSUE THREAD

**Issue by mfc** [QubesOS/qubes-issues#1776]
we need to stop using abbreviations, which are:
* confusing for new users
* bad for localization/translation

anything "sys-" should instead be "system-"
- `sys-net` -> `system-network`
- `sys-firewall` -> `system-firewall`
- `sys-usb` -> `system-usb`

**Comment by Zrubi** [comment]
Why we have to use those prefixes at all?? 
Wouldn't be more simple using:
- network
- firewall
- usb

Thos are shorter. And sys-/system is not add anything meaningful to them.

**Comment by marmarek** [comment]
Those VMs are generally something that user shouldn't (need to)
interact with in normal cases. But I agree that using name prefix for it
isn't ideal. Especially having 31 chars limit... Probably worth a tag
"system" (#865) and then other indication like separate section in Qubes
Manager ("System VMs"/"System Qubes").

-- 
Best Regards,
Marek Marczykowski-Górecki
Invisible Things Lab
A: Because it messes up the order in which people normally read text.
Q: Why is top-posting such a bad thing?

**Comment by mfc** [comment]
I like those ideas and dropping the prefixes. +1

**Comment by woju** [comment]
On Thu, Feb 25, 2016 at 01:58:05AM -0800, Marek Marczykowski-Górecki wrote:
> Probably worth a tag
> "system" (#865) and then other indication like separate section in Qubes
> Manager ("System VMs"/"System Qubes").

Tags aren't meant to do anything by ourselves, like separating something
in Qubes Manager. They are solely for the user. Probably better use
features (revamped services, still not implemented).

-- 
regards,                        _.-._
Wojtek Porczyk               .-^'   '^-.
Invisible Things Lab         |'-.-^-.-'|
                             |  |   |  |
 I do not fear computers,    |  '-.-'  |
 I fear lack of them.        '-._ :  ,-'
    -- Isaac Asimov             `^-^-_>

**Comment by axon-qubes** [comment]
Personally, I like the `sys-` tags because they cause all system-related VMs to be sorted together lexically. I also prefer `sys-` over `system-` because I find shorter names easier to visually scan at a glance (e.g., when reading through a long list of VMs), and I find `sys-` to be an intuitive and unambiguous abbreviation for `system` in the context of Qubes.

**Comment by mfc** [comment]
> Personally, I like the sys- tags because they cause all system-related VMs to be sorted together lexically

okay but "system" VMs should instead be sorted or kept in a "system vm" section of QVM based on their attribute of being "system vms", not by prefix.

**Comment by comradekingu** [comment]
Adding 'system-' to network or firewall relies on context to signify 'remote' or 'local'. Then again doing away with the prefixes and adding 'remote' or 'local' where it applies is probably the better option.

**Comment by ninavizz** [comment]
Yes, to both @mfc and @andrewdavidwong's comments, above. 

<img width="887" alt="Screen Shot 2021-09-13 at 12 26 49 PM" src="https://user-images.githubusercontent.com/8262612/133147939-41bb94e8-c005-4720-9490-d7de8cd132f6.png">

There is definitely a problem of too-long prefixes in Qubes today, introducing a cognitive burden to users when seeking to parse item names in Qubes Manager and App Menu. And yet, neither utility groups qubes by "Type" or "Role," so the only way to do that grouping is by alphabetization... which the prefixes function to achieve, as a hack to the prior problem. 

#6665 will solve for the App Menu need, which leaves open the need to introduce "Sort By Role" (vs by _Type_, which is a separate lengthy discussion) as a functionality in Qubes Manager. Which may then call into question if the API/backend already have these identifiers built-in. 

Once those are built-in and this capability is built-in to Qubes Manager, I feel we can close this ticket—as it seems the broader solution, is "Stop using prefixes as a sorting mechanism for qubes".

**Comment by ben-grande** [comment]
In general, short names are bad. Whonix templates changed from `ws/gw` to `workstation/gateway`. `dvm` is a horrible name because it is a `DispVM template`, not a `DispVM`. About changing the system qube names:

- There is some qubesdb limit on name length, it impacts tools such as salt and ansible that create qubes with prefixes and suffixes based on targeted qube name
- There is `internal` feature and `servicevm` feature. Ther former hides the qube from various tools. The latter moves the qube in the appmenu to the last tab (SERVICE).

**Comment by mfc** [comment]
yeah `dvm` is definitely something that is confusing. it doesnt reduce the abbreviation but as an intermediary step could be `dvm-template`. or we remove the `vm` part so it is `disposable-template`. so `whonix-workstation-18-disposable-template`. or `*-disp-*` to follow the current disposable conventions (which are abbreviation-based....).

i'd be interested to know the qubesdb limit, if we are anywhere near it?

it's a different place where we have an abbreviation but i think "Qubes Global Config" could be "Qubes Global Configuration".

**Comment by marmarek** [comment]
Currently qube name is limited to 31 characters...

**Comment by ben-grande** [comment]
> Currently qube name is limited to 31 characters...

Would be very nice if this was extended on the next major release, as far as I understood, it is a breaking change because of agent protocol on the domUs and dom0.
